libbeat/reader/multiline/multiline.go

```diff
@@ -163,6 +203,68 @@ func (mlr *Reader) readFirst() (reader.Message, error) {
 	}
 }
 
+func (mlr *Reader) readFirstCount() (reader.Message, error) {
```

Comment:
for the pattern matcher we need `readFirst`, because we don't want to validate against the line against the regular expression yet. But I don't think we need `readFirstCount`. We always check for the number of lines and create an event once the amount of lines have been reached.